android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -612,9 +613,14 @@ public void onClick(View v) {
                 alertDialog.show();
                 break;
             case R.id.nav_about:
+
+                SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
+                String app_name = sharedPreferences.getString(ConstantStrings.APP_NAME, "");
+                String org_description = sharedPreferences.getString(ConstantStrings.ORG_DESCRIPTION, "");
+
                 final AlertDialog aboutUs = new AlertDialog.Builder(this)
-                        .setTitle(R.string.app_name)
-                        .setMessage(R.string.about_text)
+                        .setTitle(app_name)
+                        .setMessage(Html.fromHtml(org_description).toString())
```

Comment:
@iamareebjamal
no problem. just have a look another time so that i can do all changes at once.